tests/e2e/api/src/models/orders/shared/classes.ts

```diff
@@ -0,0 +1,412 @@
+import { MetaData } from '../../shared-types';
+import { Model } from '../../model';
+import { TaxStatus } from './types';
+
+/**
+ * Order item meta.
+ */
+export class OrderItemMeta extends Model {
+	/**
+	 * The meta data the order item.
+	 *
+	 * @type {ReadonlyArray.<MetaData>}
+	 */
+	public readonly metaData: MetaData[] = [];
+}
+
+/**
+ * Order line item tax entry.
+ */
+export class OrderItemTax extends Model {
+	/**
+	 * The total tax for this tax rate on this item.
+	 *
+	 * @type {string}
+	 */
+	public readonly total: string = '';
+
+	/**
+	 * The subtotal tax for this tax rate on this item.
+	 *
+	 * @type {string}
+	 */
+	public readonly subtotal: string = '';
+}
+
+/**
+ * An order address.
+ */
+export class OrderAddress extends Model {
```

Comment:
Just thinking out loud: maybe there's an address type that would be useful in other, non-order areas as well, and the OrderAddress would then be a further specialization of that one.

Reply:
That's possible but we don't have any of the other addresses to use as a model for deciding between base and specialization. So long as the change were backward compatible we can implement the specialization as soon as we have a use case for it.